client/src/data/types/donationForm.ts

```diff
@@ -8,12 +8,40 @@ export enum ItemCondition {
     POOR = "POOR"
 }
 
-export default interface DonationForm {
+export enum ItemStatus {
+    PENDING_APPROVAL = "PENDING_APPROVAL",
+    PENDING_DROPOFF = "PENDING_DROPOFF",
+    PENDING_MATCH = "PENDING_MATCH",
+    MATCHED = "MATCHED"
+}
+
+export interface DonationFormContributionTuple {
+    donationForm: string;
+    quantity: number;
+}
+export interface DonationFormContact {
+    firstName?: string;
+    lastName?: string;
+    email?: string;
+    phoneNumber?: string;
+}
+
+export interface DonationForm {
     _id?: string;
-    age?: number;
-    condition?: ItemCondition;
-    description?: string;
+    contact?: DonationFormContact;
     name?: string;
-    quantity?: number;
     requestGroup?: RequestGroup | null;
+    description?: string;
+    quantity?: number;
+    age?: number;
+    condition?: ItemCondition;
+
+    adminNotes?: string;
+    status?: ItemStatus;
+    quantityRemaining?: number;
+
+    donatedAt?: Date;
```

Comment:
I was curious if donatedAt would just store the time of the last made match ?

Reply:
oh donatedAt represents the time when the item was dropped off by the donor to TPC